### PR TITLE
stretchly: Add version 0.21.1

### PR DESCRIPTION
**Pull request by pcrama**
This app was also in the initial #3526 but has been split out in this separate branch

### CODE REVIEW

File: bucket/stretchly.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.21.1",
+    "description": "Cross-platform open source app that reminds you to take breaks",
+    "##": "Not a portable app: settings are in $Env:AppData\\stretchly\\config.json",
+    "homepage": "https://github.com/hovancik/stretchly/",
```
Comment (Ash258) [inline bucket/stretchly.json]:
```suggestion
    "version": "0.21.1",
    "description": "Break time reminder",
    "homepage": "https://hovancik.net/stretchly/",
```

File: bucket/stretchly.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        }
+    },
+    "bin": "stretchly.exe",
+    "shortcuts": [
+        [
+            "stretchly.exe",
+            "Stretchly"
+        ]
+    ],
+    "checkver": "github",
```
Comment (Ash258) [inline bucket/stretchly.json]:
```suggestion
    "checkver": {
        "github": "https://github.com/hovancik/stretchly"
    },
```

File: bucket/stretchly.json (hunk 1 above)
Comment (pcrama) [reply bucket/stretchly.json]: I force-pushed this suggestion to my branch.

File: bucket/stretchly.json (hunk 2 above)
Comment (pcrama) [reply bucket/stretchly.json]: I force-pushed your suggestion to my branch.

File: bucket/stretchly.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    },
+    "bin": "stretchly.exe",
+    "shortcuts": [
+        [
+            "stretchly.exe",
+            "Stretchly"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/hovancik/stretchly/"
```
Comment (Ash258) [inline bucket/stretchly.json]: ...

```suggestion
        "github": "https://github.com/hovancik/stretchly"
```

Comment (pcrama) [reply bucket/stretchly.json]: Oops...

**Pull request merged**